core/src/main/java/io/grpc/internal/AbstractStream.java

```diff
@@ -162,30 +162,28 @@ public void messageRead(InputStream is) {
      * Closes this deframer and frees any resources. After this method is called, additional calls
```

Comment:
This documentation _really_ needs to be updated to note that closing may be queued behind deframing messages.